static/app/views/sharedGroupDetails/sharedGroupHeader.tsx

```diff
@@ -30,7 +30,7 @@ const SharedGroupHeader = ({group}: Props) => (
             <FeatureBadge
               type="beta"
               title={t(
-                'Sharing Performance Issues is not yet full featured and may be missing context'
+                'Not all features have been implemented for sharing Performance Issues and may be missing context.'
```

Comment:
May I suggest:
```suggestion
                'Sharing Performance Issues hasn't been implemented for all features yet and may be missing context.'
```

Reply:
I think this isn't quite right, this wording makes it sound like sharing is something each "feature" can do, where it's more that not all issue feature related to performance issues are enabled yet for shared performance issues

Reply:
Ah yeah, "shared" vs "sharing" makes a pretty big difference to the meaning of this. Perhaps:
```suggestion
                'Not all features have been implemented for shared Performance Issues and those issues may be missing context.'
```

Reply:
Thanks @imatwawana !